bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -37,50 +37,84 @@ $BallSeen
     NO --> @Stop, @LookAtBall
 
 #Positioning
-$Localization
+$LocalizationAvailable
     YES --> @GoToRolePosition, @LookAtFieldFeatures
     NO --> @GoToRelativePosition + x:2 + y:0 + t:0
 
 #PositioningReady
-$Localization
+$LocalizationAvailable
     YES --> @GoToRolePosition, @LookAtFieldFeatures
     NO --> @GoToRelativePosition + x:2 + y:0 + t:0, @StandAndWait
 
 #GoalieBehavior
-$Localization
-    YES --> $JustUnpenalized
-        YES --> #Positioning
-        NO --> $BallSeen
-            YES --> $BallDangerous
-                YES --> #GoAndKickBallMapGoal
-                NO --> @Stop, @LookAtBall
+$LocalizationAvailable
+    YES --> $LocalizationPrecision
+        HIGH --> $TimerRunning + name:reorientation
+            YES --> @EndTimer + name:reorientation, @StartTimer + name:reorientation_block + duration:%behavior/body/reorientation_block_duration
+            NO --> $JustUnpenalized
+                YES --> #Positioning
+                NO --> $BallSeen
+                    YES --> $BallDangerous
+                        YES --> #GoAndKickBallMapGoal
+                        NO --> @Stop, @LookAtBall
+                    NO --> @SearchBall
+        LOW --> $TimerEnded + name:reorientation_block
```

Comment:
Does `reorientation_block` mean that reorientation is blocked and no reorientation will happen? I initially thought it would be a block (as in "section") where reorientation is happening. Maybe you can come up with a better name?